source/configuration_manual/authentication/index.rst

```diff
@@ -1,9 +1,116 @@
+.. _authentication-authentication:
+
 ==============
 Authentication
 ==============
 
+Authentication is split into four parts:
+
+1. :ref:`Authentication mechanisms <authentication-authentication_mechanisms>`
+
+2. :ref:`Password schemes <authentication-password_schemes>`
+
+3. :ref:`Password databases (passdb) <authentication-password_databases>`
+
+4. :ref:`User databases (userdb) <authentication-user_database>`
+
+See also :ref:`authentication penalty <authentication-authentication_penalty>`
+handling for IP addresses. See also :ref:`authentication policy support <authentication-auth_policy>`
+for making policy based decisions.
+
+Authentication mechanisms vs. password schemes
+----------------------------------------------
+
+Authentication mechanisms and password schemes are often confused,
+because they have somewhat similar values. For example there is a PLAIN
+auth mechanism and PLAIN password scheme. But they mean completely
+different things.
+
+-  **Authentication mechanism is a client/server protocol**. It's about
+   how the client and server talk to each others in order to perform the
+   authentication. Most people use only PLAIN authentication, which
+   basically means that the user and password are sent without any kind
+   of encryption to the server. SSL/TLS can then be used to provide the
+   encryption to make PLAIN authentication secure.
+
+-  **Password scheme is about how the password is hashed in your
+   password database**. If you use a PLAIN scheme, your passwords are
+   stored in cleartext without any hashing in the password database. A
+   popular password scheme MD5-CRYPT (also commonly used in
+   ``/etc/shadow``) where passwords looks like
+   "``$1$oDMXOrCA$plmv4yuMdGhL9xekM.q.I/``".
+
+-  Plaintext authentication mechanisms work with ALL password schemes.
+
+-  Non-plaintext authentication mechanisms require either PLAIN password
+   scheme or a mechanism-specific password scheme.
+
+
+Authentication in Proxies and Directors
+----------------------------------------------
+
+.. Note::
+
+  that Proxy or Director already verifies the authentication (in the reference
```

Comment:
I think "that" looks a bit weird in the output? Doesn't really need to be there anyway.